layouts/_default/_markup/td-heading-self-link.html

```diff
@@ -0,0 +1,11 @@
+{{- /*
+
+    A copy of the private "_default/_markup/_td-heading-self-link.html" template
+    in Docsy's "_default/_markup/_td-render-heading.html" partial to be able to
```

Comment:
```suggestion
    in Docsy's "_default/_markup/td-render-heading.html" template, so that we can
```

I don't think it's a partial per se?

Reply:
Indeed, applied.